learn/file_handling/xml.md

```diff
@@ -5,11 +5,13 @@ XML files in Python allow for the manipulation and parsing of XML data. XML ([Ex
   
 ## Open XML File and Read Data with Python  
 
-To read data from an XML file with Python, you can use in-built XML parser module. The most commonly used libraries for parsing XML files are [lxml](https://pypi.org/project/lxml/) and ElementTree.
+To read data from an XML file with Python, you can use in-built XML parser module. In this part we will learn how to parse XML data in Python by exploring popular Python libraries.
+
+The most commonly used libraries for parsing XML files are `[lxml]`(https://pypi.org/project/lxml/) and `ElementTree`.
```

Comment:
It's not valid link. Let's keep the old version.

```
`[lxml]`(https://pypi.org/project/lxml/) 
```

Reply:
done